distributions/Dist_ih646.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+from base_distribution import BaseDistribution
+
+class Dist_ih646(BaseDistribution):
+	def __init__(self):
+		self.f_max = 30
+		self.x_min = 20
+		self.x_max = 10
+
+
+	def pdf(self, x):
+		"""This is your PDF"""
+		return x**3+np.sin(x)+np.cos(x)
+
+	def mean(self):
+		"""This is the mean of the PDF"""
+		return 15.
+
+	def std(self):
+		"""This is the standard deviation of the pdf"""
+		return 2.882767
```

Comment:
please use an equation here. Some code that depends on self.x_min and self.x_max